### PR TITLE
Adds Codeowners

### PR DESCRIPTION
**Pull request by Arrow768**
This adds a initial implementation of the codeowners file.

If you wish to be added to the file (while this PR is open, just comment below)
Later on, just make a PR that adds you to the codeowners file.

Being a codeowner comes with the expectation to review PRs that contain files covered by this in a timely manner.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Ensure to preserve the order when editing it
+# If you want to be added, make a PR that adds you to it
+# If you are added, then you are expected to review PRs in a timely manner
+
+/code/controllers/master/ @Lohikar
+/code/controllers/subsystems/supply.dm @Arrow768
+/code/datums/api.dm @Arrow768
+/code/datums/cargo.dm @Arrow768
+/code/modules/http/ @Arrow768
+/code/modules/item_worht/ @Arrow768
```
Comment (Lohikar) [inline .github/CODEOWNERS]: `item_worth`

**Comment by Lohikar** [comment]
Add me to `code/world.dm` and `code/controllers/subsystems/initialization/` as well please.

**Comment by Alberyk** [comment]
Add me to `code\modules\customitems\item_defines.dm`, please.

**Comment by skull132** [comment]
```
/code/datums/discord/
/code/datums/server_greeting.dm
/code/defines/procs/dbcore.dm
/code/modules/client/preference_setup/
/code/modules/http/
/lib/
```

plz.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# The list is ordered alphabetical
+# Ensure to preserve the order when editing it
```
Comment (Lohikar) [inline .github/CODEOWNERS]: Minor nitpick: the grammar of this line and the line above is funky.

**Pull request merged**